src/SOS/Strike/exts.h

```diff
@@ -56,6 +56,10 @@
 // printing CDA values.
 #define CDA_TO_UL64(cda) ((ULONG64)(TO_TADDR(cda)))
 
+#ifndef IMAGE_FILE_MACHINE_RISCV64
```

Comment:
This define is in shared\pal\inc\rt\ntimage.h.  Do we need it here?

Reply:
In case of Windows builds, `IMAGE_FILE_MACHINE_RISCV64` not defined (still not available in windows headers). In runtime we also have this issue and added this defenition for Windows builds into clrnt.h file (https://github.com/dotnet/runtime/blob/f28a8a72eee05e0b909a4f2cb58d66bc0f7a01fe/src/coreclr/inc/clrnt.h#L41). Initially I also move this definition into clrnt.h, unfortunately, Strike don't use (include) it and all Windows builds in CI failed, so, I moved it here.

Reply:
Yes, ntimage.h is used only for Linux/MacOS builds. It is fine here.